config/initializers/assets.rb

```diff
@@ -1,12 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.config.assets.version = "1.0"
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
Why don't we need to do this anymore?

Reply:
not 100% sure, might have been my screwup. I'll look

Reply:
I'm not sure if this works either way but the git blame is 5 years old, probably from the last time we did a rails upgrade, so I'm just gonna revert this and keep it in here.